README.md

```diff
@@ -165,7 +165,7 @@ command directly:
 
 ```shell
 $ balena fleet create cliFleet --type fincm3
-Fleet created: cliFleet (fincm3, id 1234567)
+Fleet created: slug "admin/clifleet", device type "fincm3"
 ```
 
 As can be seen, this will return the name of the fleet, its type
```

Comment:
The description in the text needs amending as well:
```
As can be seen, this will return the fleet's slug and device type.
```

Reply:
Correct, applied in the commit "apply review findings" below as well.